pkgs/development/tools/misc/editorconfig-core-c/default.nix

```diff
@@ -1,28 +1,25 @@
-{ lib, stdenv, fetchpatch, fetchFromGitHub, cmake, pcre2, doxygen }:
-
-stdenv.mkDerivation rec {
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pcre2
+, doxygen
+}:
+
+stdenv.mkDerivation (finalAttrs: {
   pname = "editorconfig-core-c";
-  version = "0.12.5";
+  version = "0.12.7";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "editorconfig";
-    repo = "editorconfig-core-c";
-    rev = "v${version}";
-    sha256 = "sha256-4p8bomeXtA+zJ3IvWW0UZixdMnjYWYu7yeA6JUwwRb8=";
+    repo = finalAttrs.pname;
```

Comment:
usage of pname is considered anti pattern https://github.com/NixOS/nixpkgs/issues/277994